src/xdp-utils.c

```diff
@@ -586,13 +586,23 @@ parse_app_info_from_flatpak_info (int pid, GError **error)
         {
           struct statfs buf;
 
+          if (statfs (root_path, &buf) != 0)
+            {
+              /**
```

Comment:
Nitpick: please remove this empty first comment line, make it similar to the comment above (`/* Access to ...`)